src/Main.java

```diff
@@ -1,58 +1,89 @@
 public class Main {
     public static void main(String[] args) {
-        int salary = 65535;
-        int total = 0;
-        int i = 0;
-        for (; total < 1_000_000; i++) {
-            total = total + total / 100;
-            total = total + salary;
-            if (i % 5 == 0) {
-                System.out.println("Месяц " + i + " итого " + total);
-            }
+        //Task 1
+        int contributions = 15_000;
+        int totalContributions = 0;
+        int month = 0;
+        while (totalContributions <= 2_459_000) {
+            totalContributions = totalContributions + totalContributions / 100;
+            totalContributions = totalContributions + contributions;
+            month = month + 1;
+            System.out.println("Месяц " + month + ", сумма накоплений составляет " + totalContributions + " рублей");
         }
-        System.out.println(total);
+        System.out.println("Итого, сумма накоплений составила " + totalContributions + " рублей, понадобилось " + month + " месяцев");
 
-        int start = 5;
+        //Task 2
+        int start = 10;
         while (start > 0) {
-            System.out.println(start);
+            System.out.print(start + " ");
             start = start - 1;
         }
+        System.out.println();
+        int finish = 1;
+        while (finish <= 10) {
+            System.out.print(finish + " ");
+            finish = finish + 1;
+        }
+        System.out.println();
 
-        int days = 10; // изначальное количество дней в Париже
+        //Task 3
+        int populationsY = 12_000_000;
+        int birthRate = 17;
+        int mortality = 8;
+        int year = 0;
+        while (year < 10) {
+            populationsY = populationsY + (birthRate - mortality) * 1000;
```

Comment:
```suggestion
            populationsY = populationsY + populationsY * (birthRate - mortality) * 1000;
```
надо умножать ещё на populationsY, потому что сказано, что на каждую 1000 человек популяции, т.е. на 9 рождается на каждую 1000, а таких тысяч - 12 000